src/main/java/com/epam/javacourses2016/task1/SolverTask1.java

```diff
@@ -11,12 +17,38 @@
 
     /**
      * Читает строки из исходного файла и сохраняет в выходной в обратном порядке.
-     * @param input Файл с входными данными.
+     *
+     * @param input  Файл с входными данными.
      * @param output Файл с выходными данными.
      * @return Список строк, прочитанных из входного файла в прямом порядке.
      */
     public List<String> reverseFile(File input, File output) {
-        //TODO: realize solver of a task
-        return null;
+        List<String>listfromfile = null;
+
+        try {
+            listfromfile = Files.readAllLines(Paths.get(input.getPath()), StandardCharsets.UTF_8);
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        FileWriter fw = null;
+
+        try {
```

Comment:
This is good try block for java 6, but since java 7 we have try-with-resource and should use it.